salt/journal/config/etc-nginx-traits.d-strict-transport-security.conf

```diff
@@ -0,0 +1,9 @@
+if ($https) {
+    add_header Strict-Transport-Security "max-age={{ 60 * 60 * 24 * 365 }}; includeSubDomains";
```

Comment:
Do we have anything non-HTTPS still? Not having this blocks adding `preload`, which fixes a problem with the header (the initial redirect).